README.md

```diff
@@ -24,32 +24,76 @@ $ composer require setono/sylius-pickup-point-plugin
 
 This command requires you to have Composer installed globally, as explained in the [installation chapter](https://getcomposer.org/doc/00-intro.md) of the Composer documentation.
 
-
-### Step 2: Enable the plugin
-
-Then, enable the plugin by adding it to the list of registered plugins/bundles
-in the `app/AppKernel.php` file of your project:
+Add the plugin to `bundles.php`.
 
 ```php
-$bundles = [
-   new \Setono\SyliusPickupPointPlugin\SetonoSyliusPickupPointPlugin(),
-];
+Setono\SyliusPickupPointPlugin\SetonoSyliusPickupPointPlugin::class => ['all' => true],
 ```
 
-### Step 3: Import routing
+### Step 2: Import routing
 
 ````yaml
 setono_sylius_pickup_point_plugin:
     resource: "@SetonoSyliusPickupPointPlugin/Resources/config/routing.yml"
 ````
 
-### Step 4: Addition of a validation group `checkout_select_shipping`
+### Step 3: Addition of a validation group `checkout_select_shipping`
```

Comment:
Why do we need this?

Reply:
I'm guessing it's to register the validator src/Resources/config/validation/Shipment.xml but I didn't make it so I don't know if it's the correct way.

Reply:
It's required